### PR TITLE
doc: mandatory tags feature

### PR DESCRIPTION
**Pull request by Shubham9t9**
# Description

### CODE REVIEW

**Review by muksUX — COMMENTED** [review]
Name of the feature is "Tag Policy" instead of "Mandatory tags"

File: docs/user-guide/global-configurations/tags-policy.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+To add tags, follow these steps:
+
+1. From the left pane, navigate to the Global Configuration section.
+2. Select Tags within the Global Configuration section.
+
+Once you are in the Tags section, locate the `Add Tag` button in the upper-right corner of the screen. Click on this button to proceed.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/add-tag.jpg)
+
+Within the Add Tag section, you will find two options for tags:
```
Comment (ashokdevtron) [inline docs/user-guide/global-configurations/tags-policy.md:16]: Within the `Add Tag` section, you will find two options for tags:

File: docs/user-guide/global-configurations/tags-policy.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+* **Suggested tags:** These tags appear as suggestions when adding tags to applications.
```
Comment (ashokdevtron) [inline docs/user-guide/global-configurations/tags-policy.md:18]: **Suggested tags**:

(colon shouldn't be bold)

File: docs/user-guide/global-configurations/tags-policy.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* **Mandatory tags:** These tags are required for applications within the selected project.
```
Comment (ashokdevtron) [inline docs/user-guide/global-configurations/tags-policy.md:19]: **Mandatory tags**:

(colon shouldn't be bold)

File: docs/user-guide/global-configurations/tags-policy.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+To create mandatory tags, choose the second option: `Mandatory tags`. This ensures that the specified tags are mandatory for the applications within the selected project.
+
+3. Next, choose the project(s) for which you want to create mandatory tags. You can select multiple projects at once.
```
Comment (ashokdevtron) [inline docs/user-guide/global-configurations/tags-policy.md:23]: Next, choose the project(s) for which you want to create mandatory tags. You can select multiple projects at once (if required).

**Review by ashokdevtron — CHANGES_REQUESTED** [review]

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**